cwrap/prototype.py

```diff
@@ -54,15 +100,26 @@ def _registerType(type_name, type_class_or_function, is_return_type=True, storag
 _registerType("long", ctypes.c_long)
 _registerType("long*", ctypes.POINTER(ctypes.c_long))
 _registerType("char", ctypes.c_char)
-_registerType("char*", ctypes.c_char_p)
-_registerType("char**", ctypes.POINTER(ctypes.c_char_p))
+if six.PY2:
+    _registerType("char*", ctypes.c_char_p)
+    _registerType("char**", ctypes.POINTER(ctypes.c_char_p))
+if six.PY3:
+    _registerType(
+        "char*",
+        CStringHelper,
+        storage_type=ctypes.c_char_p,
+        errcheck=CStringHelper.toStr)
+    # _registerType("char**", ctypes.POINTER(CStringHelper))
```

Comment:
Either remove or support, but I would hope we can deal with `char**` in Python3 as well?

Reply:
I don't think we can support it very easily, but it seem to never be used.

Reply:
Okay, so just delete the line.  Consider also the effect of deleting the corresponding line in PY2.

Reply:
I've removed it

Reply:
Okay, just leave the Py2 as it is.